prov/rxm/src/rxm.h

```diff
@@ -127,9 +127,16 @@ struct rxm_mr {
 	struct fid_mr *msg_mr;
 };
 
+struct rxm_ep_wire_proto {
+	uint8_t	ctrl_version;
+	size_t inject_size;
+	size_t sar_limit;
```

Comment:
Please use fixed sizes in wire protocol fields -- u32 or u64, not size_t.  Watch for alignment issues and add reserved fields to ensure that 32-bit and 64-bit systems have the same size structures.

Reply:
Sure, the PR has been updated accordingly.